src/tests/JIT/HardwareIntrinsics/HardwareIntrinsics_r.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsMergedTestRunnerAssembly>true</IsMergedTestRunnerAssembly>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```

Comment:
I think it's reasonable to emit compiler-generated files for the merged wrappers (as opposed to the simple wrappers where I'd be worried about tons of files being generated potentially degrading Roslyn perf) but I think it's somewhat ugly to need to explicitly put the property on each merged wrapper; could we instead tweak the scripts so that <code>EmitCompilerGeneratedFiles</code> defaults to <code>true</code> when <code>IsMergedTestRunnerAssembly</code> is set to true?